argusclient/client.py

```diff
@@ -349,8 +363,10 @@ class DashboardsServiceClient(BaseUpdatableModelServiceClient):
 
     There is no need to instantiate this directly, as it is available as :attr:`argusclient.client.ArgusServiceClient.dashboards` attribute.
     """
-    def __init__(self, argus):
-        super(DashboardsServiceClient, self).__init__(Dashboard, argus, id_path="dashboards/%s", get_all_path="dashboards")
+    def __init__(self, argus, all_dash_params=None):
+        super(DashboardsServiceClient, self).__init__(Dashboard, argus, id_path="dashboards/%s",
+                                                          get_all_path="dashboards",
+                                                          get_all_params=all_dash_params)
```

Comment:
This will change like this:
```suggestion
        super(DashboardsServiceClient, self).__init__(Dashboard, argus, id_path="dashboards/%s",
                                                          get_all_req_opts=dict(REQ_PATH="dashboards",
								                REQ_PARMS=all_dash_params))
```

Again, the same as a non-suggestion:
```
        super(DashboardsServiceClient, self).__init__(Dashboard, argus, id_path="dashboards/%s",
                                                          get_all_req_opts=dict(REQ_PATH="dashboards",
								                REQ_PARMS=all_dash_params))
```